_content/essays/impressions-of-insightful-experience_lachung-apo.md

```diff
@@ -0,0 +1,21 @@
+---
+title: "Impressions of an Insightful Experience"
+authors:
+  - "Lachung Apo"
+translator: Adam Pearcey
+subcat: poetry
+external_url: "https://www.lotsawahouse.org/tibetan-masters/lachung-apo/aho-ye"
+drive_links:
+  - "https://drive.google.com/file/d/11c_NzgNEg-_7naIlaKpFa6QYVpyvqS1q/view?usp=sharing"
+course: emptiness
+tags:
+  - perception
+  - dzogchen
+year: 2018
+pages: 1
+---
+
+> The tongue of rational argument, which clings to reality and delusion,
```

Comment:
Lets go with the first line instead in case people are searching for the poem by the first line:

```
> Aho ye! When looking at these outer appearances...
```